### DISCUSSION TITLE
is it possible to run this from an external drive?

### DISCUSSION THREAD

**Discussion by anarchy89** [lima-vm/lima#1377] [Q&A] [answered]
i dont have much space on my internal drive, is it possible to run this from an external drive?

**Comment by afbjorklund** [comment, marked as answer]
You can set the `LIMA_HOME` to another location, where there is more disk space available.

**Reply by anarchy89** [reply]
Sorry how do I do that? Is it in the documentation?

**Reply by jandubois** [reply]
I guess it is somewhat underdocumented, I only see it in https://github.com/lima-vm/lima/blob/master/docs/internal.md#lima-home-directory-lima_home

But really all you do is set `LIMA_HOME` to the directory where you want to store the instances. Ideally in your shell config file:

```
export LIMA_HOME=/Volumes/MyDrive/Lima
```

**Reply by afbjorklund** [reply]
It should probably be mentioned explicitly in the docs.
